cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    # "Blocks until the database is available"
```

Comment:
The docstring should be enclosed in triple quotes to follow Python's convention for docstrings. Change it to `"""Blocks until the database is available"""`.